crates/typst-cli/src/args.rs

```diff
@@ -15,6 +15,10 @@ pub struct CliArguments {
     /// -v = warning & error, -vv = info, -vvv = debug, -vvvv = trace
     #[clap(short, long, action = ArgAction::Count)]
     pub verbosity: u8,
+
+    /// Path to a custom CA certificate to use when making network requests.
+    #[clap(short = 'A', env = "TYPST_CERT")]
+    pub ca_certificate: Option<PathBuf>,
```

Comment:
I prefer the name `--cert` as for Deno. The shorthand `-A` is strange, I think we should skip it.